arangod/Statistics/StatisticsFeature.cpp

```diff
@@ -609,25 +601,21 @@ StatisticsFeature::StatisticsFeature(Server& server)
       if (it.second != nullptr) {
         auto const& builder = *it.second;
         auto const& stat = statIt->second;
-        std::string statName(stat[0]);
-        auto const& statType = metricType(stat[1], true);
-        if (statType == "counter") {
-          statName += "_total";
-        }
-        [[maybe_unused]] auto const& statHelp = stat[2];
-        if (builder.name() != statName) {
+        auto const name = stat[0];
+        auto const type = stat[1];
```

Comment:
```suggestion
        std::string_view name = stat[0];
        std::string_view type = stat[1];
        std::string_view help = stat[2];
```

Reply:
auto here is std::string_view

Reply:
@kvahed sorry, but that's nitpicking.

Reply:
That's not a reason of why this can't be merged, but personal preference.

Reply:
guys, take it easy. that was oversight. lgtm now